fluentd/lib/fluent-plugin-collected/test/plugin/test_in_collected_tail_monitor.rb

```diff
@@ -9,16 +9,14 @@ def setup
   end
 
   MONITOR_CONFIG = %[
+  @type tail
+     path /tmp/tmp.log, /var/log/containers/mypodname_mynamespace_mycontainername-34646d7fb38199129ab8d0e6f41833d26e1826cba92571100fd6c53904a5317e.log
```

Comment:
can we make container path as constant variable ? This is getting used later also.